rust/crd/src/lib.rs

```diff
@@ -40,6 +40,8 @@ pub struct SupersetClusterSpec {
     /// Desired Superset version
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub version: Option<String>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub statsd_exporter_version: Option<String>,
```

Comment:
`statsd_exporter_version` is optional here, but we throw an exception if it is not found. Should make it mandatory?

Reply:
Yeah that comes down to the question if we always and automatically enable monitoring?

Reply:
This will be left as it is as the documentation makes clear that this property is required (and we handle other mandatory fields - such as version - the same way).